### ISSUE TITLE
Missing Text/LABEL for DOL and OSHA Seal/Logo - Main Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#8]
The iPod we are using to test announces the DOL logo is as "Button" and goes to the DOL home page. The OSHA logo says "Button" too and goes to their home page. I would expect these to be links possibly, but tell me where they are going. The should say "United States Department of Labor website" and "OSHA - Occupational Safety and Health Administration Website". Normally, on a website I would add the word website, but wanted to have people know they are going to the website and not staying in the application, since it says these are buttons.

WCAG 2.0 - 1.3.1 - Info and Relationships - Level A

**Comment by skeeJay** [comment]
Will address.

**Comment by skeeJay** [comment]
The DOL and OSHA logos are are requirements. Though links do not apply here, because we're in an app (and textual buttons do not make sense because of space available), I can add accessibility labels so that VoiceOver announces them to accessibility users correctly.

**Comment by jfc3-dol** [comment]
Apologies for using website based terms for hidden text, when I should have been using the term accessibility LABELs.

VoiceOver is reading the accessibility LABELs correctly.

Closing Issue.
